extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -255,7 +243,8 @@ public void configurationDescriptorBuilding(
             BuildProducer<SystemPropertyBuildItem> systemProperties,
             BuildProducer<NativeImageResourceBuildItem> nativeImageResources,
             BuildProducer<HotDeploymentWatchedFileBuildItem> hotDeploymentWatchedFiles,
-            BuildProducer<PersistenceUnitDescriptorBuildItem> persistenceUnitDescriptors) {
+            BuildProducer<PersistenceUnitDescriptorBuildItem> persistenceUnitDescriptors,
+            BuildProducer<JpaEntityPerPersistenceUnitsBuildItem> jpaEntitiesPerPersistenceUnits) {
```

Comment:
I think I would name it `JpaModelPersistenceUnitMappingBuildItem`.

Also I already built something here to push it to the JPAConfig: https://github.com/quarkusio/quarkus/blob/master/extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java#L455 so I don't think we should build it again .

It's not perfect either as we can't really properly deal with XML descriptors but I think it's good enough for what we want.

Reply:
> I think I would name it `JpaModelPersistenceUnitMappingBuildItem`.
> 
> Also I already built something here to push it to the JPAConfig: https://github.com/quarkusio/quarkus/blob/master/extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java#L455 so I don't think we should build it again .
> 

Ah that's better, for some reason I missed this. I'd re-use this. 

> It's not perfect either as we can't really properly deal with XML descriptors but I think it's good enough for what we want.

In case of XML descriptors, what is the best way to deal with them? Should we use the default PU or ?

Reply:
Well, have a look if it works or not but I think you won't have the entities in the descriptors.

Frankly, I think we don't really care, at least not in this first version of the multi-persistence unit support.

Reply:
Okay.